SECURITY.md

```diff
@@ -0,0 +1,111 @@
+- Be in English, if possible.
+
+What you can expect from us:
+
+- When you choose to share your contact information with us, we commit to coordinating with you as openly and as quickly as possible.
+- Within 3 business days, we will acknowledge that your report has been received.
+- To the best of our ability, we will confirm the existence of the vulnerability to you and be as transparent as possible about what steps we are taking during the remediation process, including on issues or challenges that may delay resolution.
+- A public disclosure date is negotiated by the immudb team, the SDK developers and the bug submitter. We prefer to fully disclose the bug as soon as possible once a user mitigation or patch is available. It is reasonable to delay disclosure when the bug or the fix is not yet fully understood, the solution is not well-tested, or for distributor coordination. The timeframe for disclosure is from immediate (especially if it’s already publicly known) to a maximum of 90 business days.  
+
+We will maintain an open dialogue to discuss issues.
+
+## Guidelines
+
+Under this policy, “research” means activities in which you:
+
+- Notify us as soon as possible after you discover a real or potential security issue.
+- Make every effort to avoid privacy violations, degradation of user experience, disruption to production systems, and destruction or manipulation of data.
+- Only use exploits to the extent necessary to confirm a vulnerability’s presence. Do not use an exploit to compromise or exfiltrate data, establish persistent command line access, or use the exploit to pivot to other systems.
+- Provide us a reasonable amount of time to resolve the issue before you disclose it publicly.
+- Do not submit a high volume of low-quality reports.
+
+Once you’ve established that a vulnerability exists or encounter any sensitive data (including personally identifiable information, financial information, or proprietary information or trade secrets of any party), **you must stop your test, notify us immediately, and not disclose this data to anyone else.**
+
+## Test methods
+
+The following test methods are not authorized:
+
+- Network denial of service (DoS or DDoS) tests or other tests that impair access to or damage a system or data
+- Physical testing (e.g. office access, open doors, tailgating), social engineering (e.g. phishing, vishing), or any other non-technical vulnerability testing
+
+## Authorization
+
+If you make a good faith effort to comply with this policy during your security research, we will consider your research to be authorized, we will work with you to understand and resolve the issue quickly, and Codenotary will not recommend or pursue legal action related to your research. Should legal action be initiated by a third party against you for activities that were conducted in accordance with this policy, we will make this authorization known.
+
+## Questions

```

Comment:
Minor - each header should be surrounded by an empty line (https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md022---headings-should-be-surrounded-by-blank-lines)

```suggestion
## Questions

```